.travis.yml

```diff
@@ -1,7 +1,23 @@
-sudo: required
 services: [ docker ]
+stages:
+  - build-image
+  - name: trigger-dockerhub
+    if: branch = master AND type != pull_request
 env:
-  - DOCKERFILE=Dockerfile
```

Comment:
Why I removed the env fields here:
1. The `trigger-dockerhub` stage should happen only once, not 3 times
2. It is not so easy to provide name fields, when we have a matrix (If name field is provided they significantly improve CI interface)